demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -1131,6 +1134,47 @@ def run_command_effect(arg):
         assert modified_files_list == {'Packs/HelloWorld/Integrations/HelloWorld.yml'}
         assert modified_packs == {'HelloWorld'}
 
+    DEPRECATED_FILE_PATHS = [
+        DEPRECATED_INTEGRATION_YML,
+        SMALL_TOVERSION_INTEGRATION_YML,
+        DEPRECATED_SCRIPT_YML,
+        SMALL_TOVERSION_SCRIPT_YML,
+        SMALL_TOVERSION_PLAYBOOK_YML
+    ]
```

Comment:
instead of adding new test files -please use the test-suite to create the deprecated files.